empire.go

```diff
@@ -708,6 +760,10 @@ func (e *Empire) StreamLogs(app *App, w io.Writer, duration time.Duration) error
 
 // CertsAttach attaches an SSL certificate to the app.
 func (e *Empire) CertsAttach(ctx context.Context, app *App, cert string) error {
+	if err := authorizeApp(ctx, "CertsAttach", app); err != nil {
```

Comment:
Yeah, I'm fine with either - I think our permission list will be sufficiently short that we don't need wildcards necessarily.